libs/vm-modules/tests/unit/ml/ml_model_tests.cpp

```diff
@@ -1096,9 +1093,7 @@ TEST_F(VMModelTests, model_sequential_flatten_tensor_data)
 {
```

Comment:
add some tests for empty tensor construction and interaction (test how it fails when you assign values to it, check you can reshape it and then assign values etc.)

Reply:
Done